### PR TITLE
migrate  content from CHM

### PR DESCRIPTION
**Pull request by v-kents**

### CODE REVIEW

File: style-guide/exceptions.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+## How-To
+
+If the topic for the member you are documenting does not already contain an Exceptions section, you can insert one by placing your cursor right after the **returnValue** end tag and choosing **exceptions** from the **Tags &amp; Attributes** pane.
+
+For each exception, add an **exception** tag and insert a link to the exception type (tagged as **codeEntityReference**, *qualifyHint*=false). Follow this with **content** and **para** tags to describe the condition. Write each condition as if the word "if" precedes it. Write the description in present tense, unless past or future tense is truly necessary to convey the progression of action. In the built topic, the information is displayed in a table with Exception and Condition columns, as shown in the examples below.
+
+Fit the description to the following model:
+
+```
+*&lt;Exception&gt;* is thrown if *&lt;condition&gt;*.
```
Comment (mairaw) [inline style-guide/exceptions.md]: I think this should out of the code block

File: style-guide/fields.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Field Value Subsection
```
Comment (mairaw) [inline style-guide/fields.md]: I wanted to put the Field Value and the Property Value all in one file along with the Return Value information.

File: style-guide/fields.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+
+The Field Value subsection describes the value of a field. In DxEditor, use the Return Value section for the field value description.
```
Comment (mairaw) [inline style-guide/fields.md]: I thought I'd have a chance to explain the task. But I also wanted to remove any references to DxEditor and DxStudio as well.

File: style-guide/parameters.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
+# Parameters Subsection
+
+The Parameters subsection lists and describes each parameter in the declaration.
+
+## Guidelines
+
+DxEditor displays a Parameters section automatically, if it's required. In this section, add a description in a separate paragraph after the parameter name.
```
Comment (mairaw) [inline style-guide/parameters.md]:
```suggestion
docs.microsoft.com displays a Parameters section automatically if it's required.
```

File: style-guide/parameters.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+
+- Include an accurate description of what should be passed for the parameter. When you write this description, think about how the parameter is used by the method. 
+- Don't include a link to the parameter type, because that's added by the build, but you can specify the type in plain text (for example, "a string" for a **System.String** object). This reference should be programming language-neutral.
+- Keep the description concise, but provide enough context to make it meaningful. Don't just repeat the parameter name. For example, a *length* parameter for a operation that sets the length of a stream could be described like this: "The desired length of the current stream, in bytes." Avoid meaningless descriptions like this: "The length."
```
Comment (mairaw) [inline style-guide/parameters.md]:
```suggestion
- Keep the description concise, but provide enough context to make it meaningful. Don't just repeat the parameter name. For example, a *length* parameter for an operation that sets the length of a stream could be described like this: "The desired length of the current stream, in bytes." Avoid meaningless descriptions like this: "The length."
```

File: style-guide/parameters.md [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+- Include important information such as which enumeration values are supported, whether the value can be null, the unit of measurement, if applicable, and the default value, if one exists. For example: the *bufferSize* parameter for the **System.IO.Stream.CopyTo (Stream, Int32)** method has the following description:<br /><br />*bufferSize*<br />
+          Type: System.Int32<br />          The size of the buffer, in bytes. This value must be greater than zero. The default size is 4096
```
Comment (mairaw) [inline style-guide/parameters.md]:
```suggestion
          Type: System.Int32<br />          The size of the buffer, in bytes. This value must be greater than zero. The default size is 4096.<br />
```

File: style-guide/return.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+- Don't include a link to the return value type, because that's added by the build, but you can specify the type in plain text (for example, "a 32-bit signed integer" for a **System.Int32** object). For Boolean, enum, and array return types, follow the phrasing guidelines in [Return Value: Method Topics (DevDiv Style Guide)](#method-topics).
+- If you need to document multiple return values that cannot be described in a single paragraph, use a table with the headings "Return value" and "Description". Introduce the table with the sentence "One of the following values:".
+- Do not bury important information about the return value in Remarks. Include it in the description, but point to Remarks for a lengthier discussion if necessary.
+
+## How-To
+
+When a method returns a value, you can use DxReflector to add a Return Value section for the method. </p><p xmlns="">
```
Comment (mairaw) [inline style-guide/return.md]: let's remove this.

File: style-guide/properties.md [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+- Include measurement units (for example, "in bytes" or "in pixels") if applicable.
+- Don't include a link to the property value type in your descriptions. The build tools include this link automatically.
+- Do not bury important information about the property value in Remarks. Include it in the description, but point to Remarks for a lengthier discussion if necessary.
+
+The following table shows the boilerplate wording for property value descriptions. The wording varies according to the property type.
+
+| Property type | Wording | Examples |
+|---------------|---------|----------|
+| Class, interface, or structure | A/An/The &lt;*description, without specifying the data type*&gt;. The default is *default value*.
+  > [!NOTE]
```
Comment (mairaw) [inline style-guide/properties.md]: Notes don't work inside tables. I think we could probably do something like this to work around that:

```suggestion
  **Note:**<br />If the abstraction is not clear from the context, you can use the wording "An object that identifies/specifies/contains...". 
```

File: style-guide/properties.md [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+  > Information the user should notice even if skimmingIf the abstraction is not clear from the context, you can use the wording "An object that identifies/specifies/contains...". However, avoid this unless there is no other way to describe the property value. | Type: **System.Windows.Forms.Cursor**<br />The current default cursor.<br />Type: **System.Net.ICredentials**<br />The authentication credentials associated with this request. The default is **null**. |
```
Comment (mairaw) [inline style-guide/properties.md]:
```suggestion
  > If the abstraction is not clear from the context, you can use the wording "An object that identifies/specifies/contains...". However, avoid this unless there is no other way to describe the property value. | Type: **System.Windows.Forms.Cursor**<br />The current default cursor.<br />Type: **System.Net.ICredentials**<br />The authentication credentials associated with this request. The default is **null**. |
```

File: style-guide/summary.md [hunk 10]
```diff
@@ -0,0 +1,174 @@
+---
```
Comment (mairaw) [inline style-guide/summary.md]: You can remove the metadata section.

File: style-guide/exceptions.md [hunk 11]
```diff
@@ -0,0 +1,52 @@
...
+- Use the [Exceptions report](http://vivodina/) to determine which exceptions a specific member throws. The report also indicates whether the exceptions thrown by a member are documented or undocumented, and it allows you to tag the exceptions that you've documented. However, you should investigate each exception that is identified in the report as undocumented by examining the source code. In gathering exception information, the backend application examines nearly the complete call stack for a method. It sometimes has difficulty eliminating exceptions that are thrown farther in the call stack by a method that is passed a set of default parameters. (This might happen, for example, if a parameterless constructor passes a string argument to another constructor overload, which throws an exception if the string argument passed to it is null. In this case, the string argument can never be null, but the exception can nevertheless appear in the Exceptions report.)
```
Comment (mairaw) [inline style-guide/exceptions.md]: This is no longer true so let's remove it

```suggestion
```

File: style-guide/fields.md (hunk 2 above)
Comment (v-kents) [reply style-guide/fields.md]: Ah, sorry about that - I thought it was a typo. Fixing.

File: style-guide/properties.md (hunk 8 above)
Comment (v-kents) [reply style-guide/properties.md]: Yeah, I thought that might be a problem, but wanted to see what it looked like. Changing

File: style-guide/summary.md [hunk 12]
```diff
@@ -0,0 +1,156 @@
...
+
+### General Summary for an Overloaded Member
+
+When a class explicitly defines multiple overloads, the first overload topic appears in docs.microsoft.com with the **Overload:** preface. This preface is required by the build tools; do not change or delete it. You must write the general summary broadly enough to apply to all the individual overloads. Follow the summary wording guidelines for the particular kind of member you are documenting (for more information, see the relevant subsection under <a href="efd07549-bbec-49ab-94a5-6b88f06f0a2e.htm">Summary Section</a>), and use text that is programming language-neutral. For an example, see the table in the next section.
```
Comment (mairaw) [inline style-guide/summary.md]:
```suggestion
When a class explicitly defines multiple overloads, the first overload topic appears in docs.microsoft.com with the **Overload:** preface. This preface is required by the build tools; do not change or delete it. You must write the general summary broadly enough to apply to all the individual overloads. Follow the summary wording guidelines for the particular kind of member you are documenting (for more information, see the relevant subsection in this article), and use text that is programming language-neutral. For an example, see the table in the next section.
```

File: style-guide/summary.md [hunk 13]
```diff
@@ -0,0 +1,156 @@
...
+- [Property Topics](#property-topics)
+- [Overloaded Member Topics](#overloaded-member-topics)
+
+## Class Topics
+
+The following table provides wording guidelines, boilerplate text, and examples of Summary sections in class topics. For event-related classes, such as **EventArgs** and event-handler delegates, see [Summary: Event-Related Topics](#event-related-topics).
+
+| Item | Wording | Examples |
+|------|---------|----------|
+| General class, interface, or structure | &lt;Begin with a present-tense third-person verb.&gt; | ***DataGrid****class summary:*<br />Displays XDO data in a scrollable grid.<br />***TimeSpan****structure summary*:<br />Represents a time interval. |
```
Comment (mairaw) [inline style-guide/summary.md]: This was a problem in the original content. Would you mind adding the missing spaces here? e.g.

```suggestion
| General class, interface, or structure | &lt;Begin with a present-tense third-person verb.&gt; | ***DataGrid*** *class summary:*<br />Displays XDO data in a scrollable grid.<br />***TimeSpan*** *structure summary*:<br />Represents a time interval. |
```

**Comment by v-kents** [comment]
@mairaw Any idea why this PR isn't trying to build now? I thought I had resolved all your changes?

**Comment by mairaw** [comment]
That is indeed strange. Let me see if closing and reopening fixes that.

**Pull request merged**